cmd/nerdctl/compose_build_linux_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestComposeBuild(t *testing.T) {
+	testutil.RequiresBuild(t)
+	base := testutil.NewBase(t)
+	defer base.Cmd("builder", "prune").Run()
+
+	const dockerComposeYAML = `
+services:
+  svc0:
+    build: .
+    platform: amd64
+    ports:
+    - 8080:80
+  svc1:
+    build: .
+    platform: arm64
+    ports:
+    - 8081:80
+  svc2:
+    build: .
+    platform: linux/arm/v7
+    ports:
+    - 8082:80
+`
+	dockerfile := fmt.Sprintf(`FROM %s
+RUN uname -m > /usr/share/nginx/html/index.html
+`, testutil.NginxAlpineImage)
+
+	comp := testutil.NewComposeDir(t, dockerComposeYAML)
+	defer comp.CleanUp()
+	comp.WriteFile("Dockerfile", dockerfile)
+
+	// 1. no service args (all services are built)
+	base.ComposeCmd("-f", comp.YAMLFullPath(), "build").AssertOK()
```

Comment:
Can you assert and remove the that the images that are built present by doing nerdctl images; and nerdctl rmi

Reply:
Updated the test.